packages/html/ds/src/file-upload/file-upload.html

```diff
@@ -10,7 +10,7 @@
   {% if isError %}
     <div
       class="gi-pt-2 gi-mb-4 gi-px-4 gi-border-solid gi-border-l-lg gi-border-red-600"
-    ></div>
```

Comment:
Will this be an error? 🤔

Reply:
it's closed below with
```
{% if isError %}
</div>
{% endif %}
```
I added by mistake in the previous PR